src/components/ModalFlows/CreateProposalScreens/ActionFees.js

```diff
@@ -51,98 +51,56 @@ function ActionFeesModal({ agreement, onCreateTransaction }) {
         </span>
       </Field>
 
-      {compactMode && (
-        <div
-          css={`
-            margin-top: ${GU * 2}px;
-            weight: 100%;
-          `}
-        >
-          <img
-            css={`
-              height: ${GU * 4}px;
-              width: ${GU * 4}px;
-            `}
-            src={iconFees}
-            alt=""
-          />
-        </div>
-      )}
-
       <div
         css={`
           display: flex;
           width: 100%;
           height: ${GU * 4}px;
           margin-top: ${!compactMode ? GU * 3 : 0}px;
+          align-items: center;
         `}
       >
-        {!compactMode && (
-          <div
-            css={`
-              flex: 0.25;
-            `}
-          >
-            <img
-              css={`
-                height: ${GU * 4}px;
-                width: ${GU * 4}px;
-              `}
-              src={iconFees}
-              alt=""
-            />
-          </div>
-        )}
+        <img
+          css={`
+            height: ${GU * 4}px;
+            width: ${GU * 4}px;
+          `}
+          src={iconFees}
+          alt=""
+        />
 
-        <div
+        <h3
           css={`
-            flex: ${compactMode ? 1 : 2};
-            padding: 0 0 0 ${!compactMode ? 1.25 * GU : 0}px;
-            align-self: center;
+            font-weight: 600;
+            margin-left: 3%;
```

Comment:
Lets use px here since i think making it 8px margin should be okay 
```suggestion
            margin-left: ${GU}px;
```